### DISCUSSION TITLE
tree layout

### DISCUSSION THREAD

**Discussion by raibove** [reaviz/reaviz#86] [Q&A] [answered]
Can we create a tree layout?

I checked https://github.com/reaviz/reaflow

but I wanted to create something like this

![tree](https://user-images.githubusercontent.com/44832446/192228188-b636de77-7295-49d2-9f07-04902a8221f0.png)

**Comment by amcdnl** [comment, marked as answer]
reaflow would be the solution for this. It has some advanced layout positioning that could do this.

**Reply by raibove** [reply]
Thanks, I was able to achieve a similar layout using reaflow, just need to check how to merge the edges. I have also asked it in Q&A [here](https://github.com/reaviz/reaflow/discussions/183)
